packages/happy-dom/src/nodes/html-input-element/HTMLInputElement.ts

```diff
@@ -23,6 +23,28 @@ import IShadowRoot from '../shadow-root/IShadowRoot';
 import NodeList from '../node/NodeList';
 import EventPhaseEnum from '../../event/EventPhaseEnum';
 
+/**
+ * Return iso-week number from given date
+ *
+ * @param date Date|number.
+ * @returns Iso-week string.
+ */
+export const dateIsoWeek = (date: Date | number): string => {
```

Comment:
I think it is better if we create a separate utility. Perhaps it can be called HTMLInputDateUtility or something.

Reply:
Yes, it seems like a good idea, where would you like to put this utility in the file tree ?

Reply:
I think in `packages/happy-dom/src/nodes/html-input-element/` to start with. If we start to use it some place else we can think about moving it.